routes/sms.js

```diff
@@ -0,0 +1,33 @@
+const config = require('../config');
+const twilioClient = require('twilio')(config.twilioSID, config.twilioToken)
+const MessagingResponse = require('twilio').twiml.MessagingResponse;
+
+//Used just as an example on how to send twilio messages
+
+
+const sendTestSMS = function(req,res){
+  console.log(req.body);
```

Comment:
Was this left in intentionally?

Reply:
yes for now. I want to be able to send test messages and have an example for how to do it. It will be removed later when we have actual messaging implemented